src/storage/mongoose/mongo-store.ts

```diff
@@ -58,147 +59,147 @@ export class MongoStore implements IDataStore {
+          reject(err)
         } else {
-          const result = saveResult.toObject();
-          delete result.password;
-          delete result.searchText;
-          resolve(modelFactory.create(result));
+          const result = saveResult.toObject()
+          delete result.password
+          delete result.searchText
+          resolve(modelFactory.create(result))
         }
-      });
-    });
+      })
+    })
   }
 
   public saveMany<T extends BaseModel>(
     entities: T[],
-    modelFactory?: ModelFactory<T>,
+    modelFactory?: ModelFactory<T>
   ): Promise<T[]> {
     return new Promise((resolve, reject) => {
       this.getModel<T>(modelFactory).insertMany(entities, (err, saveResult) => {
         if (err) {
-          reject(err);
+          reject(err)
         } else {
           const result = saveResult.map((sr) => {
-            const temp = sr.toObject();
-            delete temp.password;
-            delete temp.searchText;
-            return modelFactory.create(temp);
-          });
-          resolve(result);
+            const temp = sr.toObject()
+            delete temp.password
+            delete temp.searchText
+            return modelFactory.create(temp)
+          })
+          resolve(result)
         }
-      });
-    });
+      })
+    })
   }
 
   public update<T extends BaseModel>(
     filter: LooseObject,
     dataToUpdate: LooseObject,
-    modelFactory?: ModelFactory<T>,
+    modelFactory?: ModelFactory<T>
   ): Promise<T> {
     const result = this.getModel<T>(modelFactory).findOneAndUpdate(
       filter,
       dataToUpdate,
-      { new: true },
-    );
+      { new: true }
+    )
 
     return result
       .lean()
       .exec()
       .then((r) => {
-        return modelFactory.create(r);
-      });
+        return modelFactory.create(r)
+      })
   }
 
   public toObjectId(id: string): Types.ObjectId {
-    return Types.ObjectId(id);
+    return Types.ObjectId(id)
   }
 
   public count<T extends BaseModel>(
     data?: LooseObject,
-    modelFactory?: ModelFactory<T>,
+    modelFactory?: ModelFactory<T>
   ): Promise<number> {
-    return this.getModel<T>(modelFactory).countDocuments(data).exec();
+    return this.getModel<T>(modelFactory).countDocuments(data).exec()
   }
 
   public deleteMany<T extends BaseModel>(
     filter: LooseObject,
-    modelFactory?: ModelFactory<T>,
+    modelFactory?: ModelFactory<T>
   ): Promise<DeleteResult> {
     return this.getModel<T>(modelFactory)
       .deleteMany(filter)
       .exec()
       .then((result) => {
-        return { success: true, deletedCount: result.deletedCount };
-      });
+        return { success: true, deletedCount: result.deletedCount }
+      })
   }
 
   private getModel<T extends BaseModel>(
-    modelFactory: ModelFactory<T>,
+    modelFactory: ModelFactory<T>
   ): MongoosModel<Document> {
-    if (modelFactory.getType() === typeof Account) {
-      return account;
+    if (modelFactory.getType() === typeof TodoItem) {
```

Comment:
Since you are removing support for Account model and associated functionality, please remove all such code.